tests/integration/test_backup_restore_on_cluster/test_concurrency.py

```diff
@@ -276,15 +276,10 @@ def make_backups():
     for node in nodes:
         assert_eq_with_retry(
             node,
-            f"SELECT status from system.backups WHERE id IN {ids_list} AND (status == 'CREATING_BACKUP')",
-            "",
-        )
-
-    for node in nodes:
-        assert_eq_with_retry(
-            node,
-            f"SELECT status, error from system.backups WHERE id IN {ids_list} AND (status == 'BACKUP_FAILED')",
```

Comment:
Why was it removed? We must check that backups were successful.

Reply:
It is moved to single query
https://github.com/ClickHouse/ClickHouse/pull/67027/files#diff-b0c6059a60439150d892d295856de7dcd042339e92efbf8df8276419f6791f1dR280

Reply:
When a backup fails it's better to be able to see in the test results why it happened.

Your code won't show us any errors, whereas the removed code could do that:
```
assert_eq_with_retry(
            node,
            f"SELECT status, error from system.backups WHERE id IN {ids_list} AND (status == 'BACKUP_FAILED')",
    "")
```

And your code won't show

Reply:
You are right about this